mongo.gemspec

```diff
@@ -31,4 +31,5 @@ Gem::Specification.new do |s|
   s.bindir            = 'bin'
 
   s.add_dependency 'bson', '~> 4.1'
+  s.add_dependency 'addressable'
```

Comment:
I'd rather avoid adding a dependency on another gem and stick with the core Ruby URI module instead.